<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaUtenti</class>
 <widget class="QWidget" name="VistaUtenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1132</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1132</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vista Utenti</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>1051</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="list_view"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <item>
       <widget class="QPushButton" name="ins_cliente_button">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: black;
border-radius: 20px</string>
        </property>
        <property name="text">
         <string>INSERISCI CLIENTE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ins_all_button">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: black;
border-radius: 20px</string>
        </property>
        <property name="text">
         <string>INSERISCI ALLENATORE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ins_nutr_button">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: black;
border-radius: 20px</string>
        </property>
        <property name="text">
         <string>INSERISCI NUTRIZIONISTA</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>9</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_informazioni">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;
border-radius: 15px</string>
   </property>
   <property name="text">
    <string>OTTIENI INFORMAZIONI</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>745</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1132</width>
     <height>745</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1132</width>
     <height>745</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>visulizzaListaUtenti.png</pixmap>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>button_informazioni</zorder>
 </widget>
 <resources>
  <include location="vistaListaUtenti.qrc"/>
 </resources>
 <connections/>
</ui>
